boards/xtensa/esp_wrover_kit/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
-if(CONFIG_PINMUX_ESP32)
+if(CONFIG_PINCTRL_ESP32)
```

Comment:
ditto

Reply:
Good point. I have changed it to evaluate `CONFIG_GPIO_ESP32` instead.